sources/packages/backend/apps/api/src/route-controllers/education-program/_tests_/e2e/education-program.institutions.controller.createEducationProgram.e2e-spec.ts

```diff
@@ -166,6 +169,252 @@ describe("EducationProgramInstitutionsController(e2e)-createEducationProgram", (
         assessedById: null,
         submittedById: collegeFUser.id,
         isActive: true,
+        isExpired: false,
+      }),
+    );
+  });
+
+  it("Should throw duplicate SABC code for education program when there is already an active education program with the same SABC code.", async () => {
+    // Arrange
+    const sameSabcCode = `${faker.random.alpha({ count: 3 })}9`;
```

Comment:
These random generated SABC codes can cause conflicts during the E2E tests if not under our control. We should have a single place to have them added to avoid conflicts. For the sake of this PR we can at least enforce in a hardcoded manner. Same for the other E2E using `sameSabcCode`.